lib/metaforce/login.rb

```diff
@@ -22,7 +22,12 @@ def login
           :password => password
         }
       end
-      response.body[:loginResponse][:result]
+
+      # The global override to convert the SOAP XML to camel case breaks the
```

Comment:
Where is this global override? Would it be better to just prevent it from camel-casing this use in the first place?